### ISSUE TITLE
Zip code is editable in User Info

### ISSUE THREAD

**Issue by JLewis12** [mkoza178/Subscription-Panel-Implementation#9]
The Zip Code should be locked down in the User Information fields.  Selection for offers presented to the user was made from the Zip code entered on the landing page.  Allowing a user to update the zip code in the "User info.." area could un-validate the offers available to them in the newly enter Zip code area. 

![image](https://user-images.githubusercontent.com/26151269/36262304-e840a754-121b-11e8-98ca-f217626121c0.png)

**Comment by mkoza178** [comment]
Yes that is correct, it can be edited but we also allow you to run a 2nd zip check which would block users who change their zip to an undeliverable address. Which should solve for your use case.
